app/src/main/java/movierating/exceptions/UserAlreadyExistsException.java

```diff
@@ -0,0 +1,7 @@
+package movierating.exceptions;
+
+public class UserAlreadyExistsException extends Exception {
+    public UserAlreadyExistsException(String message) {
```

Comment:
I think the argument name should be userName instead of message.

Reply:
Changed this.